docs/source/workloadConfig.rst

```diff
@@ -80,6 +80,17 @@ explicitly build this workload. This option may include arguments for the script
 However, any affects that host-init has on the resulting rootfs *will* be
 reflected in the child.
 
+bin-post
```

Comment:
Could we call this 'post-bin'? If I'm talking about it I'd say 'this script runs post-bin' not 'this script runs bin-post'.